### PR TITLE
Feat rspy130/grafana

### PR DESCRIPTION
**Pull request by eroan-marie**

### CODE REVIEW

File: inventory/sample/group_vars/all/apps.yml [hunk 1]
```diff
@@ -37,6 +37,17 @@ certmanager:
     acme:
       email: user@example.com
 
+grafana:
+  custom_plugins: []
```
Comment (nleconte-csgroup) [inline inventory/sample/group_vars/all/apps.yml]: Could you describe in a documentation how to use that custom_plugins line ? For e.g. how can I edit it to use a plugin that was developed in phase1 ?
On phase1, we had to edit this file instead : https://github.com/COPRS/infrastructure/blob/main/inventory/sample/host_vars/setup/apps/grafana.yaml

File: apps/grafana/podmonitor.yaml
Comment (nleconte-csgroup) [inline apps/grafana/podmonitor.yaml:1]: Any reason to use PodMonitor instead of ServiceMonitor ?

**Review by nleconte-csgroup — COMMENTED** [review]
@eroan-marie Could you have a look at my comments ?

File: inventory/sample/group_vars/all/apps.yml (hunk 1 above)
Comment (eroan-marie) [reply inventory/sample/group_vars/all/apps.yml]: Done

File: apps/grafana/podmonitor.yaml
Comment (eroan-marie) [reply apps/grafana/podmonitor.yaml:1]: Converted

**Review by nleconte-csgroup — CHANGES_REQUESTED** [review]
You must also add the header from now on every files, even the crds, e.g. : https://github.com/RS-PYTHON/rs-infrastructure/blob/08e063d177a8a30cb784f7af6a214b063db7246a/delete.yaml#L1-L13

**Review by nleconte-csgroup — APPROVED** [review]

**Pull request merged**